docs/content/preview/explore/change-data-capture/debezium-connector-postgresql.md

```diff
@@ -0,0 +1,1904 @@
+
+    <!-- YB specific -->
+    * `yboutput` is the plugin packaged with YugabyteDB. 
+
+    * `pgoutput` is the standard logical decoding output plug-in in YugabyteDB 10+. It is maintained by the YugabyteDB community, and used by YugabyteDB itself for logical replication. This plug-in is always present so no additional libraries need to be installed. The Debezium connector interprets the raw replication event stream directly into change events.
+
+<!-- YB note driver part -->
+* Java code (the actual Kafka Connect connector) that reads the changes produced by the chosen logical decoding output plug-in. It uses YugabyteDB’s [streaming replication protocol](#reminder), by means of the YugabyteDB JDBC driver
+
+The connector produces a change event for every row-level insert, update, and delete operation that was captured and sends change event records for each table in a separate Kafka topic. Client applications read the Kafka topics that correspond to the database tables of interest, and can react to every row-level event they receive from those topics.
+
+YugabyteDB normally purges write-ahead log (WAL) segments after some period of time. This means that the connector does not have the complete history of all changes that have been made to the database. Therefore, when the YugabyteDB connector first connects to a particular YugabyteDB database, it starts by performing a consistent snapshot of each of the database schemas. After the connector completes the snapshot, it continues streaming changes from the exact point at which the snapshot was made. This way, the connector starts with a consistent view of all of the data, and does not omit any changes that were made while the snapshot was being taken.
+
+The connector is tolerant of failures. As the connector reads changes and produces events, it records the WAL position for each event. If the connector stops for any reason (including communication failures, network problems, or crashes), upon restart the connector continues reading the WAL where it last left off. This includes snapshots. If the connector stops during a snapshot, the connector begins a new snapshot when it restarts.
+
+{{< tip title="Behaviour with Logical Decoding" >}}
+
+The connector relies on and reflects the YugabyteDB logical decoding feature, which has the following limitations:
+
+* Logical decoding does not support DDL changes. This means that the connector is unable to report DDL change events back to consumers.
+
+* Logical decoding replication slots are supported on only primary servers. When there is a cluster of YugabyteDB servers, the connector can run on only the active primary server. It cannot run on hot or warm standby replicas. If the primary server fails or is demoted, the connector stops. After the primary server has recovered, you can restart the connector. If a different YugabyteDB server has been promoted to primary, adjust the connector configuration before restarting the connector.
```

Comment:
We don't have the notion of "primary servers", "hot" or "warm standby replicas" like PostgreSQL. We use the terms Primary Clusters, ReadReplicas, and Standby Clusters (for DR case). Please update this paragraph

Reply:
This whole section is actually not applicable to us. I have removed this.